cmd/bosun/sched/check.go

```diff
@@ -198,6 +198,10 @@ func (s *Schedule) CheckUnknown() {
 				continue
 			}
 			a := s.Conf.Alerts[ak.Name()]
+			//alert doesn't exist anymore. Maybe status loaded from old db state.
```

Comment:
Rather, I'd like to understand why this bug happened before signing off on this fix.